### ISSUE TITLE
standardized CRS

### ISSUE THREAD

**Issue by andybeet** [NEFSC/NEFSC-Spatial#54]
Add a function to the package to standardize all shapefile to a user supplied CRS (with a default)

This was discussed in 2023-07-25 meeting https://docs.google.com/document/d/1_L9sUFn7oP5RDoH_qL9Gjr-iFtcj9kicAkX-cuKrJ8A/edit

**Comment by andybeet** [comment]
Is this something that we still want to do? Or should we just let a user do this themselves?
What do you think @jmhatch ?

**Comment by jmhatch** [comment]
Probably a good idea, but for simplicity I'm leaning towards storing the data in the projection the data owner used and then letting the user figure it out. As long as it's documented, I don't see any issues. Maybe just a note on the `pkgdown` site to warn users that the data may be stored in different projections, so beware?

**Comment by andybeet** [comment]
yeah ok, sounds good. i like that. How would you word that @jmhatch ? Something like

"Note: The data are made available in the projection determined by the owner of the dataset. Datasets may have different projections
"
